### PR TITLE
Update github tag action

### PR DESCRIPTION
**Pull request by katrogan**
Signed-off-by: Katrina Rogan <katroganGH@gmail.com>

# TL;DR
Signed-off-by: Katrina Rogan [katroganGH@gmail.com](mailto:katroganGH@gmail.com)

matches https://github.com/flyteorg/flytetools/pull/62

see `https://github.com/anothrNick/github-tag-action/issues/147`

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by wild-endeavor — APPROVED** [review]

**Comment by ckiosidis** [comment]
Did something change in this PR with the logging?
We see a lot of unstructured gorm logs in flyteadmin.

example

```

{"data":{"src":"handlers.go:237"},"message":"Running authentication gRPC interceptor","severity":"DEBUG","timestamp":"2022-05-02T07:56:12Z"}
{"data":{"src":"handlers.go:193"},"message":"gRPC server info in logging interceptor []method [/flyteidl.service.AdminService/ListExecutions]\n","severity":"DEBUG","timestamp":"2022-05-02T07:56:12Z"}
...
2022/05/02 07:56:12 /go/src/github.com/flyteorg/flyteadmin/pkg/repositories/gormimpl/execution_repo.go:103
[18.420ms] [rows:0] SELECT * FROM "executions" WHERE executions.execution_project = 'some-project' AND executions.execution_domain = 'staging' AND executions.phase in ('RUNNING') AND executions.started_at >= '2022-04-08T07:56:12' AND executions.started_at <= '2022-05-02T07:46:12' AND executions.state = 'EXECUTION_ACTIVE' LIMIT 100
```

**Comment by katrogan** [comment]
resolving in slack.
